sopel/coretasks.py

```diff
@@ -9,21 +9,27 @@
 # (yanovich.net)
 # Copyright © 2012, Elad Alfassa <elad@fedoraproject.org>
 # Copyright 2012-2015, Elsie Powell embolalia.com
+# Copyright 2019, Florian Strzelecki <florian.strzelecki@gmail.com>
+#
 # Licensed under the Eiffel Forum License 2.
 from __future__ import unicode_literals, absolute_import, print_function, division
 
 import logging
 from random import randint
+import collections
 import datetime
 import re
 import sys
 import time
+
 import sopel
 import sopel.module
 import sopel.tools.web
+from sopel import loader
```

Comment:
Adding this causes LGTM to [flag line R25](https://lgtm.com/projects/g/sopel-irc/sopel/snapshot/5f682a51ca40fac982830cd8429d512c35f7bb92/files/sopel/coretasks.py?sort=name&dir=ASC&mode=heatmap#x59bd4334b139ad7b:1) (`import sopel`) as a duplicate import.

At least, I assume it's this change. It couldn't really be anything else in this patch.